website/docs/reference/function-configs.md

```diff
@@ -0,0 +1,238 @@
+    # Function-specific configs are defined in the property file
+    # See functions/schema.yml examples below
+
+```
+
+</File>
+
+</TabItem>
+
+
+<TabItem value="property-yaml">
+
+<File name='functions/schema.yml'>
+
+```yaml
+version: 2
+
+functions:
+  - name: [<function-name>]
+    config:
+      # Standard configs that apply to functions
+      [database](/reference/resource-configs/database): <string>
+      [schema](/reference/resource-properties/schema): <string>
+      [alias](/reference/resource-configs/alias): <string>
+      [tags](/reference/resource-configs/tags): <string> | [<string>]
+      [meta](/reference/resource-configs/meta): {<dictionary>}
+
+```
+
+</File>
+
+</TabItem>
+
+</Tabs>
+
+### General configurations
+
+<ConfigGeneral />
+
+<Tabs
+  groupId="config-languages"
+  defaultValue="project-yaml"
+  values={[
+    { label: 'Project file', value: 'project-yaml', },
+    { label: 'Property file', value: 'property-yaml', },
+  ]
+}>
+
+<TabItem value="project-yaml">
+
+<File name='dbt_project.yml'>
+
+```yaml
+functions:
+  [<resource-path>](/reference/resource-configs/resource-path):
+    [+](/reference/resource-configs/plus-prefix)[enabled](/reference/resource-configs/enabled): true | false
+    [+](/reference/resource-configs/plus-prefix)[tags](/reference/resource-configs/tags): <string> | [<string>]
+    [+](/reference/resource-configs/plus-prefix)[database](/reference/resource-configs/database): <string>
+    [+](/reference/resource-configs/plus-prefix)[schema](/reference/resource-properties/schema): <string>
+    [+](/reference/resource-configs/plus-prefix)[alias](/reference/resource-configs/alias): <string>
+    [+](/reference/resource-configs/plus-prefix)[meta](/reference/resource-configs/meta): {<dictionary>}
+
+```
+
+</File>
+
+</TabItem>
+
+
+<TabItem value="property-yaml">
+
+<File name='functions/schema.yml'>
+
+```yaml
+version: 2
+
+functions:
+  - name: [<function-name>]
+    config:
+      [enabled](/reference/resource-configs/enabled): true | false
+      [tags](/reference/resource-configs/tags): <string> | [<string>]
+      [database](/reference/resource-configs/database): <string>
+      [schema](/reference/resource-properties/schema): <string>
+      [alias](/reference/resource-configs/alias): <string>
+      [meta](/reference/resource-configs/meta): {<dictionary>}
+
+```
+
+</File>
+
+</TabItem>
+</Tabs>
+
+## Configuring functions
+Functions are configured in YAML files, either in `dbt_project.yml` or within an individual function's YAML properties file. The function body is defined in a SQL file in the `functions/` directory.
+
+Function configurations, like model configurations, are applied hierarchically — configurations applied to a subdirectory will take precedence over configurations applied to the entire project, and configurations defined in a specific function's properties will override configurations defined in `dbt_project.yml`.
```

Comment:
this is true of all configurations - i'm not sure we need to explicitly call this out here

https://docs.getdbt.com/reference/define-configs#config-inheritance